core/src/main/java/com/sequenceiq/distrox/v1/distrox/converter/InstanceGroupV1ToInstanceGroupV4Converter.java

```diff
@@ -45,9 +44,6 @@
 
     private InstanceGroupV4Request convert(InstanceGroupV1Request source, DetailedEnvironmentResponse environment) {
         InstanceGroupV4Request response = new InstanceGroupV4Request();
-        if (InstanceGroupType.GATEWAY == source.getType() && source.getNodeCount() != 1) {
```

Comment:
Why was this changed?

Reply:
This check was moved to a place where we can check if it is a load-balancer enabled cluster or not.